lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -356,7 +356,7 @@ sub engine_workit {
     # if NEEDLES_DIR is an URL address, it means that users specify it and want to get the needles from URL.
     # In these two scenarios, doing symlink is useless and the job may incomplete because fail to do symlink.
     if (   $vars{NEEDLES_DIR}
-        && $vars{ENABLE_RELATIVE_PATH}
+        && !$vars{DISABLE_RELATIVE_PATH}
```

Comment:
I suppose the alternative would be:

```suggestion
        && ($vars{ENABLE_RELATIVE_PATH} // 1)
```

It would have the advantage of not changing the name of the setting afterwards.

Reply:
This was more an RFC but I like that you've turned it around.